Source/JavaScriptCore/API/JSValue.h

```diff
@@ -402,17 +450,59 @@ NS_CLASS_AVAILABLE(10_9, 7_0)
  */
 @property (readonly) BOOL isSymbol JSC_API_AVAILABLE(macos(10.15), ios(13.0));
 
+/*!
+@property
+@abstract Check if a JSValue is a BigInt.
+*/
+@property (readonly) BOOL isBigInt JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
 /*!
 @method
 @abstract Compare two JSValues using JavaScript's <code>===</code> operator.
 */
-- (BOOL)isEqualToObject:(id)value;
+- (BOOL)isEqualToObject:(id _Null_unspecified)value;
 
 /*!
 @method
 @abstract Compare two JSValues using JavaScript's <code>==</code> operator.
 */
-- (BOOL)isEqualWithTypeCoercionToObject:(id)value;
+- (BOOL)isEqualWithTypeCoercionToObject:(id _Null_unspecified)value;
+
+/*!
+@method
+@abstract Compares two JSValues.
+@other The JSValue to compare with.
+@result A value of JSRelationCondition, a kJSRelationConditionUndefined is returned if an exception is throw.
+@discussion The result is computed by comparing the results of <code>this == other</code>, <code>this < other</code>, and <code>this > other</code>. If either <code>this</code> or <code>other</code> is (or would coerce to) <code>NaN</code>, then the result is kJSRelationConditionUndefined.
```

Comment:
Hmm, maybe we should just say "The result is computed by comparing the results of JavaScript's <code>==</code>, <code>\<</code>, and <code>\></code> operators. If either <code>self</code> or <code>other</code> is (or would coerce to) <code>NaN</code> in JavaScript, then the result is kJSRelationConditionUndefined." Since `this` isn't a keyword in Obj-C and `self` isn't a keyword in JS.